Openvstorage/Framework documentation/structure/framework.md

```diff
@@ -0,0 +1,104 @@
+## Framework structure
+
+### Packaging
```

Comment:
Packaging contains metadata for the [Frameworkt-tools](https://github.com/openvstorage/framework-tools) packager to work it's magic.
The packaging folder exposes the different distros and their control files.

The settings.json file contains metadata about the packaging process. The source_contents value is used for the tar command to compress the files.